ci/test/resources/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM {{ service_base_image.image }}
+FROM {{ ci_utils_image.image }}
```

Comment:
`hello` just needs an image with gear and hail top. I suppose building ci_utils and using it is fine though.